ee/clickhouse/models/event.py

```diff
@@ -202,3 +201,69 @@ def determine_event_conditions(
             result += "AND event = %(event)s"
             params.update({"event": v})
     return result, params
+
+
+def get_event_count_for_team_and_period(
+    team_id: Union[str, int], begin: timezone.datetime, end: timezone.datetime
+) -> int:
+    result = sync_execute(
+        """
+        SELECT count(1) as count
+        FROM events
+        WHERE team_id = %(team_id)s
+        AND timestamp > %(begin)s
+        AND timestamp < %(end)s
+    """,
+        {"team_id": str(team_id), "begin": begin, "end": end},
+    )[0][0]
+    return result
+
+
+def get_event_count_for_team(team_id: Union[str, int]) -> int:
+    result = sync_execute(
+        """
+        SELECT count(1) as count
+        FROM events
+        WHERE team_id = %(team_id)s
+    """,
+        {"team_id": str(team_id)},
+    )[0][0]
+    return result
+
+
+def get_event_count() -> int:
```

Comment:
nit: not used anywhere, but we can keep it

Reply:
Yeah it's something we can use elsewhere